### PR TITLE
Code cleanup. Removed unused "using" directives.

### PR DESCRIPTION
**Pull request by carlosrfernandez**
Small linq queries for some methods and some code comments.

Most of the changes are unused `using` statements of some default namespaces that are always added by the editor.

I'm still getting to know the code and what it's doing. Let me know your thoughts on this. 

#2

### CODE REVIEW

File: Aspekt.Bootstrap/Bootstrap.cs [hunk 1]
```diff
@@ -123,31 +114,15 @@ private static void LoadArg(InstructionHelper ic, CustomAttributeArgument arg)
...
-                case MetadataType.TypedByReference:
-                case MetadataType.IntPtr:
-                case MetadataType.UIntPtr:
-                case MetadataType.FunctionPointer:
-                case MetadataType.Object:
-                case MetadataType.MVar:
-                case MetadataType.RequiredModifier:
-                case MetadataType.OptionalModifier:
-                case MetadataType.Sentinel:
-                case MetadataType.Pinned:
```
Comment (mvpete) [inline Aspekt.Bootstrap/Bootstrap.cs]: I'd prefer to leave these in for brevity. I'm not sure what I'm doing is right just yet. I need to read a bit more about what to do for this (if it's even possible). I'm not sure how to cherrypick and just exclude this part of the PR.

Comment (mvpete) [inline Aspekt.Bootstrap/Bootstrap.cs]: Thank you so much for the changes! If you want to keep working on the project, that's awesome. I'm aiming for functionality similar to PostSharp, but with less overhead.

Comment (carlosrfernandez) [inline Aspekt.Bootstrap/Bootstrap.cs]: I'll ammend the pull request ;)

**Comment by carlosrfernandez** [comment]
@mvpete I've added the switch labels back, and I've left them separate to the default handler. Since they're known types but not really implemented. 

Let me know your thoughts.

**Pull request merged**